examples/basic_dynamic_items.rs

```diff
@@ -0,0 +1,39 @@
+use console::style;
+
+fn main() -> std::io::Result<()> {
+    cliclack::clear_screen()?;
+
+    cliclack::intro(style(" create-app ").on_cyan().black())?;
+
+    // This is the only difference between this snippet and examples/basic.rs
+    // You can supply your items dynamically, i.e. from a database or API.
+    let items_for_select = &[
```

Comment:
@petrsapak I suggest keeping at least one variable as `vec![]` to test both approaches.
We're on the finish line! :) Thanks

Reply:
Great, I updated the code.

Reply:
Btw. I found another way using iter().fold() function. But I like the items() better.

Reply:
@petrsapak Yeah, I think fold is gonna be more obscure.